### PR TITLE
Feature: ajax filters front end

### PR DESCRIPTION
**Pull request by edbrett**
- set controllers to filter by content type, partner or tags
- change queries pushed to the URL to ? & format
- update controllers to use old masonry css (temp solution to issues with ajax rendering of masonry)
- grab filters from url on page load and set the in the filter bar

### CODE REVIEW

File: app/views/activities/index.html.erb [hunk 1]
```diff
@@ -19,9 +19,30 @@
...
+      title: 'FILTER BY',
+      placeholder: 'Filter by partner',
+      key: 'partners'
+    }
+  ] %>
+  <%= render 'shared/filters', filters: filters %>
 
   <div class="row">
     <div class="small-12 small-centered columns -fix">
+<<<<<<< HEAD
```
Comment (simaob) [inline app/views/activities/index.html.erb]: I'm guessing this is an artifact?

File: app/views/publications/index.html.erb [hunk 2]
```diff
@@ -19,9 +19,35 @@
...
+      data: @years,
+      type: 'years',
+      key: 'years'
+    }
+  ] %>
+  <%= render 'shared/filters', filters: filters %>
 
   <div class="row">
     <div class="small-12 small-centered columns -fix">
+<<<<<<< HEAD
```
Comment (simaob) [inline app/views/publications/index.html.erb]: And there are a few others here.

Comment (edbrett) [inline app/views/publications/index.html.erb]: How strange. Removed.

**Comment by edbrett** [comment]
Yes, I noticed that when I started working on it. I could not find what was causing it (Logos and social media icons). They all have shrunk and fill no longer works.

**Comment by simaob** [comment]
No longer relevant.

**Pull request closed without merging**